app/server/utils/user.py

```diff
@@ -843,7 +843,7 @@ def create_user_without_transfer_account(phone):
     :return: User
     """
     temporary_first_name = 'Unknown'
-    temporary_last_name = ''
+    temporary_last_name = 'Unknown'
```

Comment:
Perhaps a different name so we can distinguish first/last? Feel free to be creative :)